ci_framework/playbooks/05-build-operators.yml

```diff
@@ -0,0 +1,23 @@
+- name: Run pre_operator_build hooks
+  vars:
+    hooks: "{{ pre_operator_build | default([]) }}"
+  ansible.builtin.import_playbook: ./hooks.yml
+
+- hosts: "{{ cifmw_target_host | default('localhost') }}"
+  gather_facts: false
+  tasks:
+    - name: CRC registry login
```

Comment:
Maybe we should set this in rhol_crc role

Reply:
Hmmm actually, THAT may be a hook - for instance, a pre_operator_build, managing the logging. Might be a nice thing as a follow-up - I'll check with the team if anyone has some interest in implementing that part :).